test/MasterViewController.m

```diff
@@ -24,13 +28,19 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    self.org = [[Organisation alloc] initWithName:@"Corporation"];
-    [self.org addEmployeeWithName:@"Rork Smith"];
-    [self.org addEmployeeWithName:@"Rork2 Smith2"];
-    [self.org addEmployeeWithName:@"Rork3 Smith3"];
-    [self.org addEmployeeWithName:@"Rork4 Smith4"];
-    
-    self.title = @"employees";
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Organisation"];
+
+    NSArray *fetchResult = [[DatabaseController sharedInstance].context executeFetchRequest:fetchRequest error:nil];
```

Comment:
Use [DatabaseController requestResults..] instead of creating a fetchRequest directly.